### PR TITLE
[DETECTRON-2 NUCLIO] bugfix for missing python binary

### PR DESCRIPTION
**Pull request by ScriptHound**
when I tried to deploy detectron-2 model on nuclio I met /bin/sh: 1: python: not found
in intermediate container. So, I've just created a symlink and that worked

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [ ] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@ScriptHound , could you please specify how to reproduce the bug? I'm using the develop branch and can build and use detectron2 model without any issues:

<details>

```console
nmanovic@nmanovic-pc:~/Workspace/cvat$ docker logs nuclio-nuclio-pth.facebookresearch.detectron2.retinanet_r101
22.04.01 09:32:27.201                 processor (I) Starting processor {"version": "Label: 1.5.16, Git commit: ae43a6a560c2bec42d7ccfdf6e8e11a1e3cc3774, OS: linux, Arch: amd64, Go version: go1.14.3"}
22.04.01 09:32:27.201                 processor (D) Read configuration {"config": "{\n    \"metadata\": {\n        \"name\": \"pth.facebookresearch.detectron2.retinanet_r101\",\n        \"namespace\": \"nuclio\",\n        \"labels\": {\n            \"nuclio.io/project-name\": \"cvat\"\n        },\n        \"annotations\": {\n            \"framework\": \"pytorch\",\n            \"name\": \"RetinaNet R101\",\n            \"spec\": \"[\\n  { \\\"id\\\": 1, \\\"name\\\": \\\"person\\\" },\\n  { \\\"id\\\": 2, \\\"name\\\": \\\"bicycle\\\" },\\n  { \\\"id\\\": 3, \\\"name\\\": \\\"car\\\" },\\n  { \\\"id\\\": 4, \\\"name\\\": \\\"motorcycle\\\" },\\n  { \\\"id\\\": 5, \\\"name\\\": \\\"airplane\\\" },\\n  { \\\"id\\\": 6, \\\"name\\\": \\\"bus\\\" },\\n  { \\\"id\\\": 7, \\\"name\\\": \\\"train\\\" },\\n  { \\\"id\\\": 8, \\\"name\\\": \\\"truck\\\" },\\n  { \\\"id\\\": 9, \\\"name\\\": \\\"boat\\\" },\\n  { \\\"id\\\":10, \\\"name\\\": \\\"traffic_light\\\" },\\n  { \\\"id\\\":11, \\\"name\\\": \\\"fire_hydrant\\\" },\\n  { \\\"id\\\":13, \\\"name\\\": \\\"stop_sign\\\" },\\n  { \\\"id\\\":14, \\\"name\\\": \\\"parking_meter\\\" },\\n  { \\\"id\\\":15, \\\"name\\\": \\\"bench\\\" },\\n  { \\\"id\\\":16, \\\"name\\\": \\\"bird\\\" },\\n  { \\\"id\\\":17, \\\"name\\\": \\\"cat\\\" },\\n  { \\\"id\\\":18, \\\"name\\\": \\\"dog\\\" },\\n  { \\\"id\\\":19, \\\"name\\\": \\\"horse\\\" },\\n  { \\\"id\\\":20, \\\"name\\\": \\\"sheep\\\" },\\n  { \\\"id\\\":21, \\\"name\\\": \\\"cow\\\" },\\n  { \\\"id\\\":22, \\\"name\\\": \\\"elephant\\\" },\\n  { \\\"id\\\":23, \\\"name\\\": \\\"bear\\\" },\\n  { \\\"id\\\":24, \\\"name\\\": \\\"zebra\\\" },\\n  { \\\"id\\\":25, \\\"name\\\": \\\"giraffe\\\" },\\n  { \\\"id\\\":27, \\\"name\\\": \\\"backpack\\\" },\\n  { \\\"id\\\":28, \\\"name\\\": \\\"umbrella\\\" },\\n  { \\\"id\\\":31, \\\"name\\\": \\\"handbag\\\" },\\n  { \\\"id\\\":32, \\\"name\\\": \\\"tie\\\" },\\n  { \\\"id\\\":33, \\\"name\\\": \\\"suitcase\\\" },\\n  { \\\"id\\\":34, \\\"name\\\": \\\"frisbee\\\" },\\n  { \\\"id\\\":35, \\\"name\\\": \\\"skis\\\" },\\n  { \\\"id\\\":36, \\\"name\\\": \\\"snowboard\\\" },\\n  { \\\"id\\\":37, \\\"name\\\": \\\"sports_ball\\\" },\\n  { \\\"id\\\":38, \\\"name\\\": \\\"kite\\\" },\\n  { \\\"id\\\":39, \\\"name\\\": \\\"baseball_bat\\\" },\\n  { \\\"id\\\":40, \\\"name\\\": \\\"baseball_glove\\\" },\\n  { \\\"id\\\":41, \\\"name\\\": \\\"skateboard\\\" },\\n  { \\\"id\\\":42, \\\"name\\\": \\\"surfboard\\\" },\\n  { \\\"id\\\":43, \\\"name\\\": \\\"tennis_racket\\\" },\\n  { \\\"id\\\":44, \\\"name\\\": \\\"bottle\\\" },\\n  { \\\"id\\\":46, \\\"name\\\": \\\"wine_glass\\\" },\\n  { \\\"id\\\":47, \\\"name\\\": \\\"cup\\\" },\\n  { \\\"id\\\":48, \\\"name\\\": \\\"fork\\\" },\\n  { \\\"id\\\":49, \\\"name\\\": \\\"knife\\\" },\\n  { \\\"id\\\":50, \\\"name\\\": \\\"spoon\\\" },\\n  { \\\"id\\\":51, \\\"name\\\": \\\"bowl\\\" },\\n  { \\\"id\\\":52, \\\"name\\\": \\\"banana\\\" },\\n  { \\\"id\\\":53, \\\"name\\\": \\\"apple\\\" },\\n  { \\\"id\\\":54, \\\"name\\\": \\\"sandwich\\\" },\\n  { \\\"id\\\":55, \\\"name\\\": \\\"orange\\\" },\\n  { \\\"id\\\":56, \\\"name\\\": \\\"broccoli\\\" },\\n  { \\\"id\\\":57, \\\"name\\\": \\\"carrot\\\" },\\n  { \\\"id\\\":58, \\\"name\\\": \\\"hot_dog\\\" },\\n  { \\\"id\\\":59, \\\"name\\\": \\\"pizza\\\" },\\n  { \\\"id\\\":60, \\\"name\\\": \\\"donut\\\" },\\n  { \\\"id\\\":61, \\\"name\\\": \\\"cake\\\" },\\n  { \\\"id\\\":62, \\\"name\\\": \\\"chair\\\" },\\n  { \\\"id\\\":63, \\\"name\\\": \\\"couch\\\" },\\n  { \\\"id\\\":64, \\\"name\\\": \\\"potted_plant\\\" },\\n  { \\\"id\\\":65, \\\"name\\\": \\\"bed\\\" },\\n  { \\\"id\\\":67, \\\"name\\\": \\\"dining_table\\\" },\\n  { \\\"id\\\":70, \\\"name\\\": \\\"toilet\\\" },\\n  { \\\"id\\\":72, \\\"name\\\": \\\"tv\\\" },\\n  { \\\"id\\\":73, \\\"name\\\": \\\"laptop\\\" },\\n  { \\\"id\\\":74, \\\"name\\\": \\\"mouse\\\" },\\n  { \\\"id\\\":75, \\\"name\\\": \\\"remote\\\" },\\n  { \\\"id\\\":76, \\\"name\\\": \\\"keyboard\\\" },\\n  { \\\"id\\\":77, \\\"name\\\": \\\"cell_phone\\\" },\\n  { \\\"id\\\":78, \\\"name\\\": \\\"microwave\\\" },\\n  { \\\"id\\\":79, \\\"name\\\": \\\"oven\\\" },\\n  { \\\"id\\\":80, \\\"name\\\": \\\"toaster\\\" },\\n  { \\\"id\\\":81, \\\"name\\\": \\\"sink\\\" },\\n  { \\\"id\\\":83, \\\"name\\\": \\\"refrigerator\\\" },\\n  { \\\"id\\\":84, \\\"name\\\": \\\"book\\\" },\\n  { \\\"id\\\":85, \\\"name\\\": \\\"clock\\\" },\\n  { \\\"id\\\":86, \\\"name\\\": \\\"vase\\\" },\\n  { \\\"id\\\":87, \\\"name\\\": \\\"scissors\\\" },\\n  { \\\"id\\\":88, \\\"name\\\": \\\"teddy_bear\\\" },\\n  { \\\"id\\\":89, \\\"name\\\": \\\"hair_drier\\\" },\\n  { \\\"id\\\":90, \\\"name\\\": \\\"toothbrush\\\" }\\n]\\n\",\n            \"type\": \"detector\"\n        }\n    },\n    \"spec\": {\n        \"description\": \"RetinaNet R101 from Detectron2\",\n        \"handler\": \"main:handler\",\n        \"runtime\": \"python:3.8\",\n        \"resources\": {},\n        \"image\": \"cvat/pth.facebookresearch.detectron2.retinanet_r101:latest\",\n        \"targetCPU\": 75,\n        \"triggers\": {\n            \"myHttpTrigger\": {\n                \"class\": \"\",\n                \"kind\": \"http\",\n                \"name\": \"myHttpTrigger\",\n                \"maxWorkers\": 2,\n                \"workerAvailabilityTimeoutMilliseconds\": 10000,\n                \"attributes\": {\n                    \"maxRequestBodySize\": 33554432\n                }\n            }\n        },\n        \"volumes\": [\n            {\n                \"volume\": {\n                    \"name\": \"volume-1\",\n                    \"hostPath\": {\n                        \"path\": \"/home/nmanovic/Workspace/cvat/serverless/common\"\n                    }\n                },\n                \"volumeMount\": {\n                    \"name\": \"volume-1\",\n                    \"mountPath\": \"/opt/nuclio/common\"\n                }\n            }\n        ],\n        \"build\": {\n            \"image\": \"cvat/pth.facebookresearch.detectron2.retinanet_r101\",\n            \"baseImage\": \"ubuntu:20.04\",\n            \"directives\": {\n                \"preCopy\": [\n                    {\n                        \"kind\": \"ENV\",\n                        \"value\": \"DEBIAN_FRONTEND=noninteractive\"\n                    },\n                    {\n                        \"kind\": \"RUN\",\n                        \"value\": \"apt-get update \\u0026\\u0026 apt-get -y install curl git python3 python3-pip\"\n                    },\n                    {\n                        \"kind\": \"WORKDIR\",\n                        \"value\": \"/opt/nuclio\"\n                    },\n                    {\n                        \"kind\": \"RUN\",\n                        \"value\": \"pip3 install torch==1.8.1+cpu torchvision==0.9.1+cpu torchaudio==0.8.1 -f https://download.pytorch.org/whl/torch_stable.html\"\n                    },\n                    {\n                        \"kind\": \"RUN\",\n                        \"value\": \"pip3 install 'git+https://github.com/facebookresearch/detectron2@v0.4'\"\n                    },\n                    {\n                        \"kind\": \"RUN\",\n                        \"value\": \"curl -O https://dl.fbaipublicfiles.com/detectron2/COCO-Detection/retinanet_R_101_FPN_3x/190397697/model_final_971ab9.pkl\"\n                    },\n                    {\n                        \"kind\": \"RUN\",\n                        \"value\": \"ln -s /usr/bin/pip3 /usr/local/bin/pip\"\n                    }\n                ]\n            },\n            \"codeEntryType\": \"image\",\n            \"timestamp\": 1648805546\n        },\n        \"platform\": {\n            \"attributes\": {\n                \"mountMode\": \"volume\",\n                \"restartPolicy\": {\n                    \"maximumRetryCount\": 3,\n                    \"name\": \"always\"\n                }\n            }\n        },\n        \"readinessTimeoutSeconds\": 60,\n        \"securityContext\": {},\n        \"eventTimeout\": \"30s\"\n    },\n    \"PlatformConfig\": null\n}", "platformConfig": "{\n    \"kind\": \"local\",\n    \"webAdmin\": {\n        \"enabled\": true,\n        \"listenAddress\": \":8081\"\n    },\n    \"healthCheck\": {\n        \"enabled\": true,\n        \"listenAddress\": \":8082\"\n    },\n    \"logger\": {\n        \"sinks\": {\n            \"stdout\": {\n                \"kind\": \"stdout\"\n            }\n        },\n        \"system\": [\n            {\n                \"level\": \"debug\",\n                \"sink\": \"stdout\"\n            }\n        ],\n        \"functions\": [\n            {\n                \"level\": \"debug\",\n                \"sink\": \"stdout\"\n            }\n        ]\n    },\n    \"metrics\": {},\n    \"scaleToZero\": {},\n    \"autoScale\": {},\n    \"cronTriggerCreationMode\": \"processor\",\n    \"ingressConfig\": {},\n    \"kube\": {\n        \"defaultServiceType\": \"ClusterIP\"\n    },\n    \"imageRegistryOverrides\": {}\n}"}
22.04.01 09:32:27.201 cessor.healthcheck.server (I) Listening {"listenAddress": ":8082"}
22.04.01 09:32:27.201            processor.http (D) Creating worker pool {"num": 2}
22.04.01 09:32:27.201 sor.http.w1.python.logger (D) Creating listener socket {"path": "/tmp/nuclio-rpc-c93cdavm475eprgu8g6g.sock"}
22.04.01 09:32:27.201 sor.http.w0.python.logger (D) Creating listener socket {"path": "/tmp/nuclio-rpc-c93cdavm475eprgu8g70.sock"}
22.04.01 09:32:27.202 sor.http.w0.python.logger (D) Using Python wrapper script path {"path": "/opt/nuclio/_nuclio_wrapper.py"}
22.04.01 09:32:27.202 sor.http.w1.python.logger (D) Using Python wrapper script path {"path": "/opt/nuclio/_nuclio_wrapper.py"}
22.04.01 09:32:27.202 sor.http.w1.python.logger (D) Using Python handler {"handler": "main:handler"}
22.04.01 09:32:27.202 sor.http.w0.python.logger (D) Using Python handler {"handler": "main:handler"}
22.04.01 09:32:27.202 sor.http.w1.python.logger (D) Using Python executable {"path": "/usr/bin/python3"}
22.04.01 09:32:27.202 sor.http.w0.python.logger (D) Using Python executable {"path": "/usr/bin/python3"}
22.04.01 09:32:27.202 sor.http.w1.python.logger (D) Setting PYTHONPATH {"value": "PYTHONPATH=/opt/nuclio"}
22.04.01 09:32:27.202 sor.http.w1.python.logger (D) Running wrapper {"command": "/usr/bin/python3 -u /opt/nuclio/_nuclio_wrapper.py --handler main:handler --socket-path /tmp/nuclio-rpc-c93cdavm475eprgu8g6g.sock --platform-kind local --namespace nuclio --worker-id 1 --trigger-kind http --trigger-name myHttpTrigger"}
22.04.01 09:32:27.202 sor.http.w0.python.logger (D) Setting PYTHONPATH {"value": "PYTHONPATH=/opt/nuclio"}
22.04.01 09:32:27.202 sor.http.w0.python.logger (D) Running wrapper {"command": "/usr/bin/python3 -u /opt/nuclio/_nuclio_wrapper.py --handler main:handler --socket-path /tmp/nuclio-rpc-c93cdavm475eprgu8g70.sock --platform-kind local --namespace nuclio --worker-id 0 --trigger-kind http --trigger-name myHttpTrigger"}
22.04.01 09:32:27.626 sor.http.w0.python.logger (I) Wrapper connected {"wid": 0, "pid": 24}
22.04.01 09:32:27.626 sor.http.w0.python.logger (D) Waiting for start
22.04.01 09:32:27.626 sor.http.w1.python.logger (I) Wrapper connected {"wid": 1, "pid": 23}
22.04.01 09:32:27.626 sor.http.w1.python.logger (D) Waiting for start
l{"datetime": "2022-04-01 09:32:27,626", "level": "info", "message": "Init context...  0%", "with": {"worker_id": "0"}}
l{"datetime": "2022-04-01 09:32:27,626", "level": "info", "message": "Init context...  0%", "with": {"worker_id": "1"}}
Loading config /usr/local/lib/python3.8/dist-packages/detectron2/model_zoo/configs/COCO-Detection/../Base-RetinaNet.yaml with yaml.unsafe_load. Your machine may be at risk if the file contains malicious content.
Loading config /usr/local/lib/python3.8/dist-packages/detectron2/model_zoo/configs/COCO-Detection/../Base-RetinaNet.yaml with yaml.unsafe_load. Your machine may be at risk if the file contains malicious content.
l{"datetime": "2022-04-01 09:32:28,238", "level": "info", "message": "Init context...100%", "with": {"worker_id": "0"}}
l{"datetime": "2022-04-01 09:32:28,238", "level": "info", "message": "Replacing logger output", "with": {"handler_name": "default", "worker_id": "0"}}
22.04.01 09:32:28.238 sor.http.w0.python.logger (D) Started
l{"datetime": "2022-04-01 09:32:28,263", "level": "info", "message": "Init context...100%", "with": {"worker_id": "1"}}
l{"datetime": "2022-04-01 09:32:28,263", "level": "info", "message": "Replacing logger output", "with": {"handler_name": "default", "worker_id": "1"}}
22.04.01 09:32:28.263 sor.http.w1.python.logger (D) Started
22.04.01 09:32:28.263                 processor (I) Starting event timeout watcher {"timeout": "30s"}
22.04.01 09:32:28.263 .webadmin.server.triggers (D) Registered custom route {"pattern": "/{id}/stats", "method": "GET"}
22.04.01 09:32:28.263 processor.webadmin.server (D) Registered resource {"name": "triggers"}
22.04.01 09:32:28.263                 processor (W) No metric sinks configured, metrics will not be published
22.04.01 09:32:28.263                 processor (D) Starting triggers {"triggers": [{"ID":"myHttpTrigger","Logger":{},"WorkerAllocator":{},"Class":"sync","Kind":"http","Name":"myHttpTrigger","Statistics":{"EventsHandledSuccessTotal":0,"EventsHandledFailureTotal":0,"WorkerAllocatorStatistics":{"WorkerAllocationCount":0,"WorkerAllocationSuccessImmediateTotal":0,"WorkerAllocationSuccessAfterWaitTotal":0,"WorkerAllocationTimeoutTotal":0,"WorkerAllocationWaitDurationMilliSecondsSum":0,"WorkerAllocationWorkersAvailablePercentage":0}},"Namespace":"nuclio","FunctionName":"pth.facebookresearch.detectron2.retinanet_r101"}]}
22.04.01 09:32:28.263            processor.http (I) Starting {"listenAddress": ":8080", "readBufferSize": 16384, "maxRequestBodySize": 33554432, "cors": null}
22.04.01 09:32:28.263 processor.webadmin.server (I) Listening {"listenAddress": ":8081"}
22.04.01 09:32:28.263                 processor (D) Processor started
22.04.01 09:34:50.056 sor.http.w0.python.logger (I) Run retinanet-R101 model {"worker_id": "0"}
```

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Bug: Incorrect point deletion with keyboard shortcut (<https://github.com/openvinotoolkit/cvat/pull/4420>)
 - some AI Tools were not sending responses properly (<https://github.com/openvinotoolkit/cvat/issues/4432>)
 - Unable to upload annotations (<https://github.com/openvinotoolkit/cvat/pull/4513>)
+- bugfix for missing python binary while nucleo deployment (<https://github.com/openvinotoolkit/cvat/pull/4532>)
```
Comment (nmanovic) [inline CHANGELOG.md:32]:
```suggestion
- bugfix for missing python binary while nuclio deployment (<https://github.com/openvinotoolkit/cvat/pull/4534>)
```

**Comment by Marishka17** [comment]
Hi, @ScriptHound, thanks for the contribution! Are you still going to contribute to us? If so, could you please resolve conflicts in this PR?

**Comment by bsekachev** [comment]
Looks like the fix is already in develop, anyway, thank you for the contribution!

**Pull request closed without merging**